examples/demo_qemu/wireguard_peer/peer2/wg0.conf

```diff
@@ -0,0 +1,9 @@
+[Interface]
+PrivateKey = uNTn0y0vYYPp8Y0RFPM+2ytVjgcZ0WBmYsf4LEOPrkg=
```

Comment:
Could you add a warning comment here, same as [here](https://github.com/XmasHackathon2023/esp_wireguard/pull/3/files#r1760990648)

Reply:
Done